triggerbot/tree_watcher.py

```diff
@@ -103,7 +109,7 @@ def failure_trigger(self, branch, rev, builder):
                               ' need to trigger it' % (builder, rev))
                 return
 
-            for entry in TreeWatcher.builder_blacklist:
+            for entry in self.get_excluded_jobs(branch, rev):
                 if entry in builder:
```

Comment:
I think this whole loop can be replaced with ```if entry in self.get_excluded_jobs(branch, rev)```. ```if entry in builder:``` was for the substring match, it isn't what we mean anymore.